### PR TITLE
Scale-out set up for local dev environment + quick manual

### PR DESCRIPTION
**Pull request by Sindica**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: docs/setup-guide/scale-out-local-dev-setup.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+1. Two Tenant Partitions
+
+1. Single Resource Partition (2/19/2021)
+
+1. HA proxy
+
+## Prerequsite
+
+1. 3 dev box (tested on ubuntu 16.04), 1 for RP, 2 for TPs. Record ip as TP1_IP, TP2_IP, RP_IP
```
Comment (yb01) [inline docs/setup-guide/scale-out-local-dev-setup.md:13]: note the TP and proxy can be put into one box, RPs needs to be separated into different boxes

File: hack/arktos-up-scale-out-poc.sh [hunk 2]
```diff
@@ -0,0 +1,609 @@
+#!/usr/bin/env bash
```
Comment (yb01) [inline hack/arktos-up-scale-out-poc.sh:1]: do we have a similar script already to reuse ?

**Review by yb01 — APPROVED** [review]
/lgtm.
this is to our own code-sharing branch. let's get this in and we can keep working on improving it.

File: docs/setup-guide/scale-out-local-dev-setup.md (hunk 1 above)
Comment (Sindica) [reply docs/setup-guide/scale-out-local-dev-setup.md:13]: Right now, this manual is for internal reference only. Will need more test and rephrase if we need it in master. Also only for 2TP/1RP right now.

File: hack/arktos-up-scale-out-poc.sh (hunk 2 above)
Comment (Sindica) [reply hack/arktos-up-scale-out-poc.sh:1]: This script is a copy over from Peng's changes. He would work on merging it into master and maximize code reuse. We will pick up his changes from master once it is merged.

**Pull request merged**